targets/TARGET_Cypress/TARGET_PSOC6/TARGET_CY8C63XX/TARGET_MCU_PSOC6_M0/system_psoc63_cm0plus.c

```diff
@@ -0,0 +1,636 @@
+/***************************************************************************//**
+* \file system_psoc63_cm0plus.c
+* \version 2.10
+*
+* The device system-source file.
+*
+********************************************************************************
+* \copyright
+* Copyright 2016-2018, Cypress Semiconductor Corporation.  All rights reserved.
+* Copyright 2017-2018, Future Electronics
+ */* SPDX-License-Identifier: Apache-2.0
```

Comment:
```suggestion
 * SPDX-License-Identifier: Apache-2.0
```

I think the '*/' might cause some issues here.

Reply:
It most definitely will. Line 1 has /* and matching */ and then another /* which originally was being matched by the */ on line 11 but then there is an extra */ on line 12.

Reply:
I have fixed this typo with the latest commit.